### PR TITLE
fix(widget): remove favorite tooltip for widget

### LINKED ISSUE #4289: Rewrite Favourites tokens tooltip in widget
1. Open the widget integrator
2. Open a token list on the Swap form
3. Hover a mouse to the Favourites tokens tooltip

**AR**: it says that a user can edit the list in the Tokens page (however, there is no possibility to do this in the widget)
image

**ER**: several options: 
1. remove the tooltip
2. rewrite it for widget: something like 'Most popular tokens to trade' or so

### PR DESCRIPTION
**Pull request by alfetopito**
# Summary

Fixes #4289

Remove favourite tooltip for widget, as it's not possible to edit them.

| Regular/Before | Widget now |
|-|-|
| image | image |

# To Test

1. Open widget
2. Open token selector
* There should be no tooltip in the favorite tokens header
3. Open CoW Swap
* There should be a tooltip in the favorite tokens header

### CODE REVIEW

**Review by shoom3301 — APPROVED** [review]

**Review by shoom3301 — APPROVED** [review]

File: apps/widget-configurator/src/app/configurator/hooks/useWidgetParamsAndSettings.ts [hunk 1]
```diff
@@ -11,8 +11,8 @@ const getBaseUrl = (): string => {
   if (isLocalHost) return 'http://localhost:3000'
   if (isDev) return 'https://dev.swap.cow.fi/'
   if (isVercel) {
-    const prKey = window.location.hostname.replace('widget-configurator-git-', '').replace('-cowswap.vercel.app', '')
-    return `https://swap-dev-git-${prKey}-cowswap.vercel.app`
+    // const prKey = window.location.hostname.replace('widget-configurator-git-', '').replace('-cowswap.vercel.app', '')
+    return `https://swap-dev-git-fix-4289favorite-tooltip-for-widget-cowswap.vercel.app`
```
Comment (alfetopito) [inline apps/widget-configurator/src/app/configurator/hooks/useWidgetParamsAndSettings.ts:15]: Just for testing.

This PR's URL breaks the pattern making the widget not work.

Revert before merging.

**Comment by alfetopito** [comment]
> Hey @alfetopito , could you please check the widget preview build? Looks like something is broken there: image

Temporarily fixed, you can test it now (it's an issue with this PR only).

**Review by elena-zh — APPROVED** [review]
Looks good to me!

File: apps/widget-configurator/src/app/configurator/hooks/useWidgetParamsAndSettings.ts (hunk 1 above)
Comment (alfetopito) [reply apps/widget-configurator/src/app/configurator/hooks/useWidgetParamsAndSettings.ts]: Reverted

**Pull request merged**